src/commands/test.js

```diff
@@ -90,3 +141,109 @@ function getPlatform(logs) {
 
   throw new Error(`Unsupported platform: ${type} ${arch} ${majorVersion}`)
 }
+
+function runDocker(coverage_opt, datasource, version_opt, latest_version, print) {
+  // Remove binary-install-raw binaries, because docker has permission issues
+  // when building the docker images
+  fs.rmSync("node_modules/binary-install-raw/bin", { force: true, recursive: true });
+
+  let dir = 'tests/.docker';
+
+  if (!fs.existsSync(dir)){
+    fs.mkdirSync(dir, { recursive: true });
+  }
+
+  try {
+    fs.writeFileSync(`${dir}/Dockerfile`, dockerfile(version_opt, latest_version))
+    print.info('Successfully generated Dockerfile.');
+  } catch (error) {
+    print.info('A problem occurred while generating the Dockerfile. Please attend to the errors below:');
+    print.info(chalk.red(error));
```

Comment:
Did you check if this looked better than `print.error`? It probably already uses `chalk.red` under the table 🕵️‍♀️

Reply:
Yes, `print.error` will be better. It prints the backtrace as well which I guess will be better for debugging